### ISSUE TITLE
使用自己的数据

### ISSUE THREAD

**Issue by zrr1991** [cfzd/Ultra-Fast-Lane-Detection#160]
parser.add_argument('config', default= 'configs/culane.py',help = 'path to config file')
    #parser.add_argument('config', help='path to config file')
    parser.add_argument('--local_rank', type=int, default=0)

    parser.add_argument('--dataset', default = None, type = str)
    parser.add_argument('--data_root', default = None, type = str)
    parser.add_argument('--epoch', default = 30, type = int)
    parser.add_argument('--batch_size', default = 4, type = int)
    parser.add_argument('--optimizer', default = 'SGD', type = str)
    parser.add_argument('--learning_rate', default = 0.01, type = float)
    parser.add_argument('--weight_decay', default = 5e-4, type = float) #None
    parser.add_argument('--momentum', default = 0.937, type = float)
    parser.add_argument('--scheduler', default = None, type = str)
    parser.add_argument('--steps', default = None, type = int, nargs='+')
    parser.add_argument('--gamma', default = None, type = float)
    parser.add_argument('--warmup', default = None, type = str)
    parser.add_argument('--warmup_iters', default = None, type = int)
    parser.add_argument('--backbone', default = None, type = str)
    parser.add_argument('--griding_num', default = None, type = int)
    parser.add_argument('--use_aux', default = None, type = str2bool)
    parser.add_argument('--sim_loss_w', default = None, type = float)
    parser.add_argument('--shp_loss_w', default = None, type = float)
    parser.add_argument('--note', default = None, type = str)
    parser.add_argument('--log_path', default = None, type = str)
    parser.add_argument('--finetune', default = None, type = str)
    parser.add_argument('--resume', default = None, type = str)
    parser.add_argument('--test_model', default = 'configs/culane_18.pth', type = str)
    parser.add_argument('--test_work_dir', default = None, type = str)
    parser.add_argument('--num_lanes', default = None, type = int)

大神，我想用您训练好的模型跑自己的图像，以上我自己设置的，除了‘config’和'--test_model'，其他是随便设置的，还有很多None不会设置，运行后给我报的错如下所示，还让我设置

usage: demo.py [-h] [--local_rank LOCAL_RANK] [--dataset DATASET]
               [--data_root DATA_ROOT] [--epoch EPOCH]
               [--batch_size BATCH_SIZE] [--optimizer OPTIMIZER]
               [--learning_rate LEARNING_RATE] [--weight_decay WEIGHT_DECAY]
               [--momentum MOMENTUM] [--scheduler SCHEDULER]
               [--steps STEPS [STEPS ...]] [--gamma GAMMA] [--warmup WARMUP]
               [--warmup_iters WARMUP_ITERS] [--backbone BACKBONE]
               [--griding_num GRIDING_NUM] [--use_aux USE_AUX]
               [--sim_loss_w SIM_LOSS_W] [--shp_loss_w SHP_LOSS_W]
               [--note NOTE] [--log_path LOG_PATH] [--finetune FINETUNE]
               [--resume RESUME] [--test_model TEST_MODEL]
               [--test_work_dir TEST_WORK_DIR] [--num_lanes NUM_LANES]
               config
demo.py: error: the following arguments are required: config

请问大神我应该如何操作呢，输入自己的图像看效果，我需要修改哪些东西呢
